erpnext/accounts/doctype/sales_invoice/sales_invoice.py

```diff
@@ -147,6 +148,12 @@ def validate_fixed_asset(self):
 					elif asset.status in ("Scrapped", "Cancelled", "Sold"):
 						frappe.throw(_("Row #{0}: Asset {1} cannot be submitted, it is already {2}").format(d.idx, d.asset, asset.status))
 
+	def validate_item_cost_centers(self):
+		for item in self.get("items",None):
+			cc = frappe.get_doc("Cost Center",item.cost_center)
+			if cc.company != self.company:
+				frappe.throw(_("Row #{0} : Cost Center <strong>{1}</strong> does not belong to company <strong>{2}</strong>").format(item.idx, item.cost_center, self.company))
```

Comment:
Don't use html tag inside message, it will create problem while translating. Instead use frappe.bold(item.cost_center)`.

Reply:
no space before `:`